samples/CosmosDbIntegtation/WriteAroundSamples/ListsSample.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using StackExchange.Redis;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Redis.Samples
+{
+    public record ListData
+    (
+        string id,
+        List<string> value
+    );
+    public static class ListsSample
+    {
+        //Redis Cache primary connection string from local.settings.json
+        public const string redisConnectionString = "redisConnectionString";
+        private static readonly IDatabase s_redisDb = ConnectionMultiplexer.Connect(Environment.GetEnvironmentVariable(redisConnectionString)).GetDatabase();
+
+
+        //CosmosDB Endpoint from local.settings.json
+        public const string CosmosDBConnectionString = "CosmosDBConnectionString";
+
+        //Uses the key of the user's choice and should be changed accordingly
+        public const string key = "userListName";
+
+        /// <summary>
+        /// This function is triggered by changes to a specified CosmosDB container. It retrieves a list of items that have been modified or added 
+        /// to the container and adds them to a Redis cache. The function converts each item's collection of values into an array and pushes the array to the Redis cache.
+        /// </summary>
+        /// <param name="readOnlyList">An IReadOnlyList of ListData objects representing the items that have been modified or added to the CosmosDB container.</param>
+        /// <param name="log">An ILogger object used for logging purposes.</param>
+        [FunctionName("CosmosToRedis")]
+        public static void Run([CosmosDBTrigger(
```

Comment:
Nit: Maybe we can give this a similar name to the function names in the other two scenarios? Something like `WriteAroundListTrigger` ?

Reply:
Yes, I did. If you're also talking about line 34, that's the name of the trigger so I cant change that part

Reply:
It seems like that can be changed in the other files right? I might not have all context, can you explain why it cannot be changed? Where is it defined?